### PR TITLE
feat(autok3s): add --k3s-install-script flag

### PR DESCRIPTION
**Pull request by Jason-ZW**
#177

Signed-off-by: Jason-ZW <zhenyang@rancher.com>

### CODE REVIEW

File: pkg/cluster/cluster.go [hunk 1]
```diff
@@ -30,10 +30,6 @@ import (
 	"k8s.io/kubectl/pkg/scheme"
 )
 
-var (
-	DefaultScript = "http://rancher-mirror.cnrancher.com/k3s/k3s-install.sh"
```
Comment (JacieChao) [inline pkg/cluster/cluster.go:34]: If the user didn't set `--k3s-install-script`, we could use this value as the default value in each provider.

**Review by JacieChao — COMMENTED** [review]
We still need to add a `--mirror` param, so users in China could use mirrors in China to install k3s.

File: pkg/cluster/cluster.go (hunk 1 above)
Comment (Jason-ZW) [reply pkg/cluster/cluster.go:34]: Already set a default value on each provider

**Review by JacieChao — APPROVED** [review]
LGTM

**Pull request merged**